pypowsybl/network.py

```diff
@@ -224,7 +237,7 @@ def forecast_distance(self) -> _datetime.timedelta:
 
     def __str__(self) -> str:
         return f'Network(id={self.id}, name={self.name}, case_date={self.case_date}, ' \
-               f'forecast_distance={self.forecast_distance}, source_format={self.source_format})'
+               f'forecast_distance={self.forecast_distance}, source_format={self.source_format}, elements={self.objects})'
```

Comment:
If you want to display up-to-date information, you need to retrieve the number of elements from here:
if we want to do this, we should implement a specific call to the java backend, otherwise it will greatly hurt the performance of `str` to gather all the dataframes (may grow to tenths of seconds on a large network).

Reply:
Yes i want to dynamically update this info.
How do you suggest we calll java backend

Reply:
can you help me a bit in understanding where should i call java backend